apps/ai-image-generator/backend/scripts/validate-actions.js

```diff
@@ -0,0 +1,33 @@
+const manifest = require('../../contentful-app-manifest.json');
+
+const validateActions = () => {
+  const requiredProperties = ['id', 'path'];
```

Comment:
I lack understanding on why we don't have to check for the entryFile

Reply:
It's a great question. From what i can tell this is not needed. It's only need for the previous `build-actions.js` script to work (since the entry file value is provided to esbuild).